test/index.html

```diff
@@ -11,16 +11,16 @@
 <body>
   <script>
     WCT.loadSuites([
-      'vaadin-combo-box-properties.html',
-      'toggling-dropdown.html',
-      'overlay-position.html',
-      'selecting-items.html',
-      'form-input.html',
-      'filtering.html',
-      'keyboard.html',
-      'scrolling.html',
-      'adaptive-overlay.html',
-      'aria.html'
+      // 'vaadin-combo-box-properties.html',
```

Comment:
Yes, these were commented out while trying to get the tests pass in Win10/IE11 in SauceLabs. Tests still fail, which I need to look into tomorrow. After that I'll add all tests back to the test suite. :smile: